### PR TITLE
Implement Cupertino switch and slider demos

### PR DESCRIPTION
**Pull request by rami-a**
Closes https://github.com/material-components/material-components-flutter-gallery/issues/98
Closes https://github.com/material-components/material-components-flutter-gallery/issues/99

Simulator Screen Shot - iPhone 11 Pro - 2019-12-18 at 14 31 24
Simulator Screen Shot - iPhone 11 Pro - 2019-12-18 at 14 31 28

### CODE REVIEW

**Comment by clocksmith** [comment]
Can you change the Column to spaceBetween?

LGTM

**Comment by rami-a** [comment]
@clocksmith done

File: gallery/gallery/lib/demos/cupertino/cupertino_slider_demo.dart [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                          .demoCupertinoSliderContinuous(
+                        _value.toStringAsFixed(1),
+                      ),
+                    ),
+                  ),
+                ],
+              ),
+              Column(
+                mainAxisSize: MainAxisSize.min,
+                children: <Widget>[
```
Comment (RedBrogdon) [inline gallery/gallery/lib/demos/cupertino/cupertino_slider_demo.dart]: nit: Don't need the `<Widget>` here. As long as you have different kinds of widget in the list, Dart will assume a type of `List<Widget>`.

Comment (rami-a) [reply gallery/gallery/lib/demos/cupertino/cupertino_slider_demo.dart]: Done, I think I just missed this one.

**Pull request merged**